test/jdk/javax/swing/JComboBox/DisabledComboBoxFontTestAuto.java

```diff
@@ -0,0 +1,176 @@
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+
+import javax.imageio.ImageIO;
+import javax.swing.DefaultListCellRenderer;
+import javax.swing.JComboBox;
+import javax.swing.SwingUtilities;
+import javax.swing.UIManager;
+import javax.swing.UnsupportedLookAndFeelException;
+
+import static java.awt.image.BufferedImage.TYPE_INT_ARGB;
+
+public class DisabledComboBoxFontTestAuto {
+    private static JComboBox combo, combo2;
+    private static BufferedImage enabledImage, disabledImage, enabledImage2, disabledImage2;
+    private static Path testDir;
+    private static String lafName;
+    private static int COMBO_HEIGHT, COMBO_WIDTH, COMBO2_HEIGHT, COMBO2_WIDTH;
+
+    private static void createCombo() {
+        combo = new JComboBox();
+        combo.addItem("Simple JComboBox");
+        combo.setRenderer(new DefaultListCellRenderer());
+        combo2 = new JComboBox();
+        combo2.addItem("Simple JComboBox");
+        COMBO_WIDTH = (int) combo.getPreferredSize().getWidth();
+        COMBO_HEIGHT = (int) combo.getPreferredSize().getHeight();
+        COMBO2_WIDTH = (int) combo2.getPreferredSize().getWidth();
+        COMBO2_HEIGHT = (int) combo2.getPreferredSize().getHeight();
+        combo.setSize(COMBO_WIDTH, COMBO_HEIGHT);
+        combo2.setSize(COMBO2_WIDTH, COMBO2_HEIGHT);
+    }
+
+    private static void paintCombo() {
+        combo.setEnabled(true);
+        enabledImage = new BufferedImage(COMBO_WIDTH, COMBO_HEIGHT, TYPE_INT_ARGB);
+        Graphics2D graphics2D = enabledImage.createGraphics();
+        combo.paint(graphics2D);
+        graphics2D.dispose();
+        combo.setEnabled(false);
+        disabledImage = new BufferedImage(COMBO_WIDTH, COMBO_HEIGHT, TYPE_INT_ARGB);
+        graphics2D = disabledImage.createGraphics();
+        combo.paint(graphics2D);
+        graphics2D.dispose();
+        combo2.setEnabled(true);
+        enabledImage2 = new BufferedImage(COMBO2_WIDTH, COMBO2_HEIGHT, TYPE_INT_ARGB);
+        graphics2D = enabledImage2.createGraphics();
+        combo2.paint(graphics2D);
+        graphics2D.dispose();
+        combo2.setEnabled(false);
+        disabledImage2 = new BufferedImage(COMBO2_WIDTH, COMBO2_HEIGHT, TYPE_INT_ARGB);
+        graphics2D = disabledImage2.createGraphics();
+        combo2.paint(graphics2D);
+        graphics2D.dispose();
+    }
+
+    private static void testMethod() throws IOException {
+        ImageIO.write(enabledImage, "png", new File(testDir
+                + "/" + lafName + "Enabled.png"));
+        ImageIO.write(disabledImage, "png", new File(testDir
+                + "/" + lafName + "Disabled.png"));
+        ImageIO.write(enabledImage2, "png", new File(testDir
+                + "/" + lafName + "EnabledDLCR.png"));
+        ImageIO.write(disabledImage2, "png", new File(testDir
+                + "/" + lafName + "DisabledDLCR.png"));
+        System.out.println("DIR: " + testDir);
```

Comment:
Is this needed? I know it is just a test but please remove the debug output when not needed.

Reply:
Removed the message. I added it to double check the saved images' file path, but didn't remove it before uploading.